src/platform/forms-system/src/js/definitions/bankAccount.js

```diff
@@ -1,3 +1,4 @@
+// Non functional change to see if tests failed.
```

Comment:
@SebVortex it seems that our team is only being pulled in because of these comments as reviewers. Are they needed? They don't seem to provide much context and I am wondering if they are carry over from past PRs

Reply:
Yeah, you're right. There was some trouble with the automation not running any tests. I'll get rid of this and the other one.